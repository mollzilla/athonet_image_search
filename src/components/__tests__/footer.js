import React from "react"
import renderer from "react-test-renderer"

import Footer from "../footer"

describe("Footer", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Footer style={{
        marginTop: `2rem`,
      }} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})