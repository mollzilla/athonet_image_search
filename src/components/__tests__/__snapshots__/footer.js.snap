// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  style={
    Object {
      "marginTop": "2rem",
    }
  }
>
  Â© 
  2021
  , Built with
   
  <a
    href="https://github.com/mollzilla"
  >
    Gatsby by Mollzilla.
  </a>
  <br />
  <a
    href="https://github.com/mollzilla"
  >
    Find out why they call me Mollzilla.
  </a>
</footer>
`;
